airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java

```diff
@@ -123,8 +123,6 @@ public ConnectionRead createConnection(final ConnectionCreate connectionCreate)
         .withStatus(ApiPojoConverters.toPersistenceStatus(connectionCreate.getStatus()));
     if (connectionCreate.getResourceRequirements() != null) {
       standardSync.withResourceRequirements(ApiPojoConverters.resourceRequirementsToInternal(connectionCreate.getResourceRequirements()));
-    } else {
-      standardSync.withResourceRequirements(workerConfigs.getResourceRequirements());
```

Comment:
Removing this line because we should not be setting the worker config resource reqs as the connection-level resource reqs. This causes the connector definition resource reqs to be ignored since the connection-level reqs take higher precedence, and the logic to default to the worker resource reqs lives in the DefaultJobCreator.

Reply:
Makes sense!